compute/instances/api-cli/cli-cheatsheet.mdx

```diff
@@ -0,0 +1,104 @@
+---
+meta:
+  title: Scaleway Instances CLI Cheatsheet
+  description: This page shows you the basic CLI commands to use Scaleway Instances.
+content:
+  h1: Scaleway Instances CLI Cheatsheet
+  paragraph: This page shows you the basic CLI commands to use Scaleway Instances.
+tags: instance
+categories: 
+  - compute
+  - instance-dev
+---
+
+This page shows the most common Scaleway CLI commands to use Instances.
```

Comment:
Not sure about "to use" (and in the meta descriptions) sounds ever so slightly odd? Maybe "This page shows the most common Scaleway CLI commands for creating and managing Scaleway Instances"?